blues/ncmc.py

```diff
@@ -345,15 +339,15 @@ def create_normalSystem(self, coord_file, top_file):
 #        self.dummy_integrator = openmm.openmm.LangevinIntegrator(temperature, friction, timestep)
 #        self.md_simulation = openmm.app.simulation.Simulation(topology=self.normalsystem.topology, system=self.normalsystem.system, integrator=self.md_integrator)
 #        self.dummy_simulation = openmm.app.simulation.Simulation(topology=self.normalsystem.topology, system=self.normalsystem.system, integrator=self.dummy_integrator)
-   
```

Comment:
@sgill2 - if these spaces are coming from you (or from you, @nathanmlim ), can you set your editor to strip trailing spaces? (Harass me on Slack if you don't know how). It's best to all use the same settings for editing text files so GitHub doesn't show changes when in fact there are no changes.